code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -209,8 +209,12 @@
 /datum/reagent/water/holywater/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
 	..()
 	if(ishuman(M)) // Any location
-		if(M.mind && cult.is_antagonist(M.mind) && prob(10))
-			cult.remove_antagonist(M.mind)
+		if(iscultist(M))
+			if(prob(10))
+				cult.offer_uncult(M)
+			if(prob(2))
+				var/obj/effect/spider/spiderling/S = new /obj/effect/spider/spiderling(M.loc)
```

Comment:
`m.loc` is not safe, unfortunately. Besides lockers, spiderlings can end up spawning in places including mechs, mob holders, cyborg chargers, and alien stomachs.